did_resolver_sov/src/reader/vdr_reader.rs

```diff
@@ -22,10 +23,18 @@ impl TryFrom<LedgerPoolConfig> for ConcreteAttrReader {
         let request_submitter = Arc::new(IndyVdrSubmitter::new(ledger_pool));
         let request_signer = Arc::new(BaseWalletRequestSigner::new(wallet.clone()));
         let response_parser = Arc::new(ResponseParser::new());
+        let cacher_config = InMemoryResponseCacherConfig {
```

Comment:
My comment here is out of PR scope, just realized something and I'll drop it here. 

I think overally, all of these components should be injectable into `did_resolver_sov`, rather than constructing new instances here. When we are going to consume this crate from `aries-vcx`, chances are we already have instances we construct here.
It would also enable custom combination of trait implementations (including the cache and its setup)

Reply:
Wholeheartedly agree, it's just that it is easier to it like this for now. This can be initialization can be easily modified to take a config object later.